frontend/src/components/selectOptionsReset.js

```diff
@@ -13,6 +14,15 @@ const selectOptionsReset = {
       height: 'auto',
     };
   },
+  groupHeading: (provided) => ({
+    ...provided,
+    fontWeight: 'bold',
+    fontFamily: 'SourceSansPro',
```

Comment:
this isn't being applied, I say just remove it

Reply:
Its needed to bold the drop down label.

Reply:
removed.